site/config/search.php

```diff
@@ -82,12 +82,128 @@
 			'release-35'
 		]
 	],
+	'loupe' => [
```

Comment:
This feels a bit weird as we have on the top level all config for Algolia but then this one key that defines all options for Loupe, some of them very similar to other top-level options. I would suggest to create separate files for these, `config/search.algolia.php` and `config/search.loupe.php`.